perllib/FixMyStreet/Cobrand/UKCouncils.pm

```diff
@@ -480,6 +480,16 @@ sub contact_email {
     return $self->feature('contact_email') || $self->next::method();
 }
 
+sub do_not_reply_email {
+    my $self = shift;
+    return $self->feature('do_not_reply_email') || $self->next::method();
+}
+
+sub verp_email_domain {
+    my $self = shift;
+    return $self->feature('verp_email_domain');
```

Comment:
Why does the method above need the extra `$self->next::method()` but this one doesn't?

Reply:
There is a Default.pm entry for do_not_reply_email, it's not an optional hook used with call_hook, whereas this one has been implemented as a hook so doesn't have a default.